azure/template.json

```diff
@@ -428,7 +425,7 @@
                                 },
                                 {
                                     "name": "ApprenticeshipsApiSubscriptionKey",
-                                    "value": "[parameters('apprenticeshipsApiSubscriptionKey')]"
+                                    "value": "[reference(concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', parameters('sharedApimResourceGroup'), '/providers/Microsoft.ApiManagement/service/', parameters('sharedApimName'), '/subscriptions/', variables('functionAppNames')[0]), '2019-01-01').primaryKey]"
```

Comment:
Is referencing the first function app in the functionAppNames array intended or should it be referencing a different function app with each iteration of the copy loop?  If it should always the same function app it might be easier to read and less prone to breaking in the future if the function app name was explicit rather than indexing into the array

Reply:
It should always be that specific function app as that is the name of the subscription that gets created elsewhere in the ARM template. I could replace both instances with `concat(variables('resourceNamePrefix'),'-apc-fa')` if that would make it more clearer?